codegen/src/main/java/software/amazon/awssdk/codegen/poet/PoetExtension.java

```diff
@@ -25,11 +25,18 @@
 /**
  * Extension and convenience methods to Poet that use the intermediate model.
  */
-public class PoetExtensions {
+public class PoetExtension {
```

Comment:
Checkstyle suddenly failed due to plural name (guess it's because I made changes in this class)